.github/workflows/main.yml

```diff
@@ -0,0 +1,19 @@
+on: push
+name: CI
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install node
+      uses: actions/npm@master
+      with:
+        node-version: 12
```

Comment:
We may as well also add 8, 10, and 12 here if that's what we're also testing in Travis (see sample starter: https://github.com/actions/starter-workflows/blob/master/ci/node.js.yml#L12)